frontend/rudrama-devi-kakatiya/rudrama-devi.css

```
@@ -0,0 +1,573 @@
+    --rudrama-bg-alt: #160f08;
+    --rudrama-card-bg: rgba(30, 20, 10, 0.78);
+    --rudrama-card-border: rgba(234, 179, 8, 0.22);
+
+    --rudrama-text: #fef9c3;
+    --rudrama-text-dim: #fef08a;
+    --rudrama-text-muted: #facc15;
+
+    --rudrama-focus: #facc15;
+    --rudrama-shadow: 0 10px 30px -10px rgba(0, 0, 0, 0.65);
+
+    --font-heading: 'Playfair Display', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)